app/javascript/mastodon/features/account_gallery/components/media_item.js

```diff
@@ -152,7 +124,12 @@ export default class MediaItem extends ImmutablePureComponent {
     return (
       <div className='account-gallery__item' style={{ width, height }}>
         <a className='media-gallery__item-thumbnail' href={status.get('url')} onClick={this.handleClick} title={title} target='_blank' rel='noopener noreferrer'>
-          <canvas width={32} height={32} ref={this.setCanvasRef} className={classNames('media-gallery__preview', { 'media-gallery__preview--hidden': visible && loaded })} />
+          <Blurhash
+            hash={attachment.get('blurhash')}
+            className={classNames('media-gallery__preview', {
+              'media-gallery__preview--hidden': visible && loaded,
+            })}
+          />
```

Comment:
Also missing `dummy` attribute I guess

Reply:
There were no previous usage of `useBlurhash`, should I add it still?

Reply:
Seems like it was an oversight.